share/completions/mkbundle.fish

```diff
@@ -0,0 +1,56 @@
+function __get_mkbundle_cross_args
```

Comment:
again, please namespace this as `__fish_mkbundle_cross_args`. This avoids collision with user functions, and communicates where the function is coming from.